packages/plugin-dom-layout/src/DomLayout.ts

```diff
@@ -0,0 +1,109 @@
+import { JWPlugin, JWPluginConfig } from '../../core/src/JWPlugin';
+import { Parser } from '../../plugin-parser/src/Parser';
+import { Renderer } from '../../plugin-renderer/src/Renderer';
+import JWEditor, { Loadables } from '../../core/src/JWEditor';
+import { Layout } from '../../plugin-layout/src/Layout';
+import { DomTemplate, DomLayoutEngine, DomTemplateSource } from './ui/DomLayoutEngine';
+import {
+    DomZonePosition,
+    LayoutEngineConfig,
+    componentReference,
+} from '../../plugin-layout/src/LayoutEngine';
+import { Dom } from '../../plugin-dom/src/Dom';
+import { VSelectionDescription, VSelection } from '../../core/src/VSelection';
+import { DomSelectionDescription } from '../../plugin-dom-editable/src/EventNormalizer';
+import { ZoneDomRenderer } from './ui/ZoneDomRenderer';
+import { ZoneDomParser } from './ui/ZoneDomParser';
+import { VElement } from '../../core/src/VNodes/VElement';
+import { VNode } from '../../core/src/VNodes/VNode';
+import { ZoneIdentifier, ZoneNode } from '../../plugin-layout/src/ZoneNode';
+
+export interface DomLayoutConfig extends JWPluginConfig {
```

Comment:
I don't see the use of having 4 differents ways to configure the layout.

Reply:
?